homeassistant/components/tractive/__init__.py

```diff
@@ -316,6 +337,27 @@ def _send_wellness_update(self, event: dict[str, Any]) -> None:
             TRACKER_WELLNESS_STATUS_UPDATED, event["pet_id"], payload
         )
 
+    def send_health_overview_update(self, event: dict[str, Any]) -> None:
+        """Handle health_overview events from Tractive API."""
+        # The health_overview response can be at root level or wrapped in 'content'
+        # Handle both structures for compatibility
+        data = event.get("content", event)
+
+        activity = data.get("activity", {})
+        sleep = data.get("sleep", {})
+
+        payload = {
+            ATTR_DAILY_GOAL: activity.get("minutesGoal"),
+            ATTR_MINUTES_ACTIVE: activity.get("minutesActive"),
+            ATTR_MINUTES_DAY_SLEEP: sleep.get("minutesDaySleep"),
+            ATTR_MINUTES_NIGHT_SLEEP: sleep.get("minutesNightSleep"),
+            # Calm minutes can be used as rest indicator
+            ATTR_MINUTES_REST: sleep.get("minutesCalm"),
+        }
+        self._dispatch_tracker_event(
+            TRACKER_WELLNESS_STATUS_UPDATED, data["petId"], payload
```

Comment:
We should use different prefix for this event. And sensors `minutes_active`, `minutes_rest`, `daily_goal`, `minutes_day_sleep` and `minutes_night_sleep` should use this prefix. Otherwise we will get a `KeyError`.
```suggestion
            TRACKER_HEALTH_OVERVIEW_STATUS_UPDATED, data["petId"], payload
```

Reply:
fixed